catalog/opensearch/catalog-opensearch-endpoint/src/main/java/org/codice/ddf/opensearch/endpoint/OpenSearchEndpoint.java

```diff
@@ -276,13 +276,19 @@ private void addSpatialFilter(
     if (StringUtils.isNotBlank(geometry)) {
       LOGGER.trace("Adding SpatialCriterion geometry: {}", geometry);
       query.addGeometrySpatialFilter(geometry.trim());
-    } else if (StringUtils.isNotBlank(bbox)) {
+    }
+
+    if (StringUtils.isNotBlank(bbox)) {
```

Comment:
This whole method 'feels' odd. Passing in the 'query' with parameters to be added in a void method seems 'side-effecty'. I would suggest refactoring this to at least return the query, but better to create one via a copy constructor or even a builder.

Reply:
while this is good feedback, these comments should be considered ✏️